Neos.Cache/Classes/Frontend/PsrFrontend.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace Neos\Cache\Frontend;
+
+/*
+ * This file is part of the Neos.Cache package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cache\Exception\PsrInvalidArgumentException;
+use Neos\Cache\Psr\PsrCacheItem;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * A frontend that implements the CacheItemPoolInterface from the PSR-6 specification.
+ */
+class PsrFrontend extends VariableFrontend implements CacheItemPoolInterface
+{
+    /**
+     * A list of items still to be persisted.
+     *
+     * @var array
+     */
+    protected $deferredItems = [];
+
+    /**
+     * Returns a Cache Item representing the specified key.
+     *
+     * @param string $key
+     * @return CacheItemInterface
+     * @throws PsrInvalidArgumentException
+     */
+    public function getItem($key)
+    {
+        if (!$this->isValidEntryIdentifier($key)) {
+            throw new PsrInvalidArgumentException('"' . $key . '" is not a valid cache entry identifier.', 1514738649629);
+        }
+
+        $rawResult = $this->backend->get($key);
+        if ($rawResult === false) {
+            return new PsrCacheItem($key, false);
+        }
+
+        $value = ($this->useIgBinary === true) ? igbinary_unserialize($rawResult) : unserialize($rawResult);
+        return new PsrCacheItem($key, true, $value);
+    }
+
+    /**
+     * Returns a traversable set of cache items.
+     *
+     * @param string[] $keys
+     * @return array
+     * @throws PsrInvalidArgumentException
```

Comment:
This method actually doesn't throw an exception. I know the interface tells otherwise, but the comment block seems wrong that way.